cmd/ssh.go

```diff
@@ -284,3 +456,89 @@ func (cmd *SSHCmd) startServices(ctx context.Context, devPodConfig *config.Confi
 		}
 	}
 }
+
+// setupGPGAgent will forward a local gpg-agent into the remote container
+// this works by using cmd/agent/workspace/setup_gpg
+func (cmd *SSHCmd) setupGPGAgent(
```

Comment:
if you have an idea, you can split that function - it's a bit longish :)

Reply:
Yes it's a bit on the long side, but I don't feel like it's doing out-of-scope things, it's doing that one single thing
which happens to be a bit long :joy: